requirements.txt

```diff
@@ -22,3 +22,4 @@ verboselogs==1.7
 websockets==6.0
 yappi==0.98
 dataclasses; python_version < '3.7'
+plyvel==1.0.5
```

Comment:
remove leveldb in requirements.txt

Reply:
leveldb supported in key_value_store_leveldb. I resolve this comment.

Reply:
> remove leveldb in requirements.txt

Please see key_value_store_leveldb.py, test_key_value_store.py, key_value_store_factory.py
KeyValueStore still support leveldb library.
We can use leveldb library as KeyValueStore backend library if we want.